flask_iiif/signals.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Flask-IIIF
+# Copyright (C) 2014, 2015 CERN.
+#
+# Flask-IIIF is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Flask IIIF Signals."""
+
+from blinker import Namespace
+
+_signals = Namespace()
+
+# Before request
+iiif_before_process_request = _signals.signal('iiif-before-process-request')
```

Comment:
Also, in the commit log message, "NEW adds signals in the restful api" would produce release note message that wouldn't be very informative to prospective users.  Perhaps expand the phrase little a bit, like: "Adds new signals to REST API that permit to do foo and bar."